src/Analyzers/Core/Analyzers/UseAutoProperty/AbstractUseAutoPropertyAnalyzer.cs

```diff
@@ -13,7 +15,10 @@
 namespace Microsoft.CodeAnalysis.UseAutoProperty
 {
     internal abstract class AbstractUseAutoPropertyAnalyzer<
-        TPropertyDeclaration, TFieldDeclaration, TVariableDeclarator, TExpression> : AbstractBuiltInCodeStyleDiagnosticAnalyzer
+        TPropertyDeclaration,
+        TFieldDeclaration,
+        TVariableDeclarator,
+        TExpression> : AbstractBuiltInCodeStyleDiagnosticAnalyzer
```

Comment:
This is rough. It would be more readable like this:

```csharp
internal abstract class AbstractUseAutoPropertyAnalyzer<TPropertyDeclaration, TFieldDeclaration, TVariableDeclarator, TExpression>
    : AbstractBuiltInCodeStyleDiagnosticAnalyzer
```

Reply:
wrapped type parameters are used a fair bit in the codebase.

Reply:
I'm only referring to this location, which changed as part of this pull request from a more readable form to a less readable form.